tools/flang2/flang2exe/bih.h

```diff
@@ -119,6 +119,12 @@ typedef struct {
       unsigned unroll : 1;     /* bih is a loop to be fully unrolled */
       unsigned unroll_cnt : 1; /* bih has a user-specified unroll factor */
       unsigned nounroll : 1;   /* bih is a loop that must not be unrolled */
+      unsigned vectorlength_scalable : 1; /* bih is a loop that must be 
+                                                vectorized width scalable
+                                              */
+      unsigned vectorlength_enabled : 1;  /* bih is a loop that must be 
```

Comment:
This comment is not quite right. Suggest "bih is a loop with a vectorlength set"